be/src/storage/rowset/column_writer.cpp

```diff
@@ -590,7 +590,7 @@ Status ScalarColumnWriter::finish_current_page() {
 
     _push_back_page(page.release());
 
-    if (is_nullable() && _opts.adaptive_page_format) {
+    if (is_nullable()) {
```

Comment:
adaptive_page_format will be false, I think this code block is always false?

Reply:
adaptive_page_format always be true.
https://github.com/StarRocks/starrocks/blob/da87fc5cd2b7dfdf1ac4c1445f445566ed91ce83/be/src/storage/rowset/segment_writer.cpp#L123